android/README.md

```diff
@@ -0,0 +1,33 @@
+# How to Set Up the App with Android Studio
+
+1. Pull the latest version of the source code from the master branch (for stable) or a branch that starts with android/ (for unstable but more features)
+2. Download Android Studio and open it up
+3. Click on open existing project
+4. Navigate to the android folder in the git repository and choose it
+5. Wait for android studio to sync with the project and download any tools that are needed to fully sync
```

Comment:
Spell this line out in a little more detail. Talk about how they have to download the android sdk for the specific version.

Reply:
Android sdk is automatically prompted by Android Studio to download if needed

Reply:
I know. But explicitly write it out in case they don't read the notification.